enhancements/multi-arch/heterogeneous-architecture.md

```diff
@@ -0,0 +1,266 @@
+---
+title: heterogeneous-architecture-clusters
+authors:
+  - "@Prashanth684"
+reviewers:
+  - "@bparees"
+  - "@derekwaynecarr"
+  - "@jupierce"
+  - "@sdodson"
+  - "@wking"
+  - "@LalatenduMohanty"
+approvers:
+  - "@bparees"
+  - "@derekwaynecarr"
+creation-date: 2022-01-17
+last-updated: 2022-01-17
+status: implementable
+---
+
+# heterogeneous-architecture-cluster-support
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for tech preview and GA
+- [ ] User-facing documentation is created in [Openshift-docs](https://github.com/Openshift/Openshift-docs/)
+
+## Summary
+
+This enhancement describes the [plan][OCPPLAN-4577] to support provisioning and upgrades of heterogeneous architecture Openshift clusters. A heterogeneous cluster is a cluster with nodes running on different architectures with a homogeneous control plane and worker node pools supporting any combination of architectures (eg: x86 control plane + ARM worker nodes).
+The node pools within the cluster are homogeneous. This document describes the support needed for heterogeneous architecture clusters in phases. There will be three major phases with basic support in phase 1 which is targeted for 4.11 and the other phases which address gaps beyond the 4.11 time frame. The targeted platforms in the initial phases are Azure
+and AWS with support for x86 and ARM architectures.
+
+![alt heterogeneous cluster](heterogeneous-cluster.png)
+
+## Motivation
+
+The motivation for supporting heterogeneous architecture clusters is so users can take advantage of cost savings on architectures like ARM.
```

Comment:
another motivation is to be able to colocate components that require different architectures, within the same cluster (avoiding cross-cluster latency and failure modes)

Reply:
added.